### ISSUE TITLE
Add class or wrapping div to iframe?

### ISSUE THREAD

**Issue by mikestecker** [remark-embedder/core#25]
Would it be possible to add a config setting that would allow for class names on the iFrame, or, better yet, a wrapping `div` with a custom `className`?

**Comment by mikestecker** [comment]
@kentcdodds I love that idea. This gives greater flexibility on adding custom HTML for each provider/transformer.

It would also be great if that function could also get the provider information from https://github.com/remark-embedder/transformer-oembed also. Something to the effect of:

```js
function handleHTML(html, {url, config}) {
  if (!isUrlIWantToChange(url, config)) return html

  if (config.provider.provider_name === 'YouTube') {
    return `<div class="youtube aspect-w-16 aspect-h-9">${html}</div>`
  }

  return `<div class="general-wrapper">${html}</div>`
}
```

I can try working on it but my skills aren't the greatest. It might be better for something more experienced than myself to handle this.

**Comment by kentcdodds** [comment]
If you get stuck, just push what you have in a PR (use `git commit` with the `--no-verify` if you need to) and I can help you out. Rather than having the provider added to the config, maybe there can be an additional option provided. I haven't thought much about it, but go ahead and try to make it work the way you think you'd like it and we can work through the API.

**Comment by mikestecker** [comment]
@kentcdodds just submitted PR. Feel free to review 👍🏻

**Comment by mikestecker** [comment]
Closed with #26
